src/orga/governance.rst

```diff
@@ -86,8 +86,11 @@ preferable. Sensitive voting items are ones where the results of the
 vote should remain private to the voters after the vote has occurred.
 Sensitive votes should take place on a secure anonymous voting platform
 (such as `Polys <polys.me>`_ or
-`the Helios voting system <https://vote.heliosvoting.org/>`_) in order to
-retain anonymity.
+`the Helios voting system <https://vote.heliosvoting.org/>`_) in order to retain
```

Comment:
Let's remove Helios. There should be only one voting platform to avoid confusion.

Reply:
Happy to.  That does expand the scope of this change, though.  If that's a sticking point for enough people, I'm also happy to leave it as-is.

Reply:
While I agree that it is kind of out of the scope part of the confusion started b/c, in helios, we only get a link to vote via e-mail. So maybe let's include that here and remove it. (Not even sure why people decided to keep it.)

Reply:
I think we had previously made changes so that we'd be platform agnostic to voting and not force us into a single platform.  CC @mariusvniekerk

Reply:
I'm going to revert that change.  I don't want to make this already-charged topic any more fraught.